### PR TITLE
Add gram giver example in C

### PR DESCRIPTION
**Pull request by akiramenai**

### CODE REVIEW

File: c/example-8-gramgiver/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+#### Notes
+Sends a message with `value` encoded in body the the public method `getGrams` of the contract with a given address.
+It's assumed that `getGrams` takes a single uint64 value.
+
+### Method `getGrams` (`giver.c`).
+#### Input values
+##### Argument `value`
+* ABI type: `uint64`
+* C type: `unisnged`
```
Comment (BorisI) [inline c/example-8-gramgiver/README.md]: "unisnged" - typo?

**Comment by BorisI** [comment]
Please add comments explaining purpose of ACCEPT() and persistent memory handling

**Comment by akiramenai** [comment]
We agreed that we first create a manual on writing contracts in C, then return back on what and how should be explained in the code.

File: c/example-8-gramgiver/README.md (hunk 1 above)
Comment (akiramenai) [reply c/example-8-gramgiver/README.md]: Fixed

**Pull request merged**